testing/supporting_docs/tools_to_find_selectors.md

```diff
@@ -0,0 +1,54 @@
+## Finding Appium Selectors
+
+### Appium Inspector
+
+When a developer installs “Appium Desktop” on their machine, not only are they installing the server 
+and GUI tooling to configure and control it, they also install a tool called appium inspector which 
+can be used to interrogate applications to find selectors.  Once the server is started clicking on 
+the “Start Inspector Session” button in the application will bring up the session configuration 
+screen.  
+
+“Desired Capabilities” are required to be entered into the configuration settings before starting 
+the inspector, and they can also be saved for use in future sessions with the same application. 
+The mandatory capabilities are “platformName”, “deviceName” and “app”, which provide information 
+such as Android or iOS device, the name of the emulator or simulator, and the path to the binary 
```

Comment:
@twnolan consistency should be persisted with the word 'android' across the whole document, e.g. starting with either lower or upper case.